packages/noya-utils/src/Platform.ts

```diff
@@ -0,0 +1,31 @@
+export enum PlatformName {
+  Web = 'Web',
+  Native = 'Native',
+  Electron = 'Electron',
+}
+
+export const Platform = {
+  isNative: false,
+  isWeb: false,
+  isElectron: false,
+};
+
+export function setPlaform(platformName: PlatformName) {
```

Comment:
Is this something that could be inferred somehow instead of having to set it manually?

E.g. we could export a `currentPlatform` different value from `.ts` and `.native.ts`